src/abstractdatatable/abstractdatatable.jl

```diff
@@ -659,17 +663,6 @@ unique!(dt)  # modifies dt
 """
 (unique, unique!)
 
-function nonuniquekey(dt::AbstractDataTable)
```

Comment:
Something which should have been removed in #17? Better do this in another PR then.